### ISSUE TITLE
Probably hard enhancement requests

### ISSUE THREAD

**Issue by CCGLLC-knc** [Dimencia/DU-Industry-Tool#22]
First, 0.8.1 is really cool.  Great work.  I really love the Production List tool.

Two long term enhancement request, and one mid term one:

1) First the easy one:  Please add, perhaps on Github, some documentation on how to set ore prices.

2) Long term - requested as an aid to growing up as an industrialist
    A)  Optionally add production times to Pure's, Products, etc.  This should, ideally, including both industry and nanopack times for those things that can be produced in the nanopack.  Goal of this request is to help starting industrialist to prioritize which industries they should add first, with the obvious goal of reducing their final time to manufacturing.  This is a feature of du-craft.net (so not critical, but given that tool is EOL, would be nice-to-have).  As an example, for a Basic Mining Unit L, Basic Pipe is shown as taking over 3 hours, and Basic Hydraulics over 2 hours.  Most everything else is under 30 minutes.  Given this information, my first industry would be a metalworks to make Basic Pipe and my Second a metalworks to make Basic Hydraulics.  Note that nanopack times for those things that can be crafted in one are oddly lower than industrial times, but of course, you only have a single nanopack (as of RC1.0).  I'm requesting this optionally since it might clutter up the screen a bit.
   B)  Optionally add a depreciation cost for the industries used.  This could become complicated, but perhaps some (settable?) defaults could be used such as 10,000 units of production for Tier 1 items, 2500 units for production of Tier 2... 10 units for production of Tier 5 units.  To be clear this would be the sum of the depreciated cost of each industry used to make the requested product and would complicate the math quite a bit.  The potential delay in a response and that some may consider their factories fully depreciated is why I'm asking for this to be optional.  Again, using the Basic Mining Unit L as a simple example, the cost of (4) refiners for the Tier 1 ores (no schematic cost since they are Tier 1), (4) smelters, (1) electronic (connector), 1 3d printer (fixation), etc. would be calculated.  This feature would go a long way towards helping establish "fair" pricing for products.  Something critical in the early days of the game.  It might also help us argue with NQ if any bots are selling items below true manufacturing cost.  One nice feature would be to be able to set a default "mark up" price and have that added to the calculation.  E.g. 20%

**Comment by tobitege** [comment]
Hello again,
we developers hate doing documentation, but you probably already know that. 🤣 
Thanks for those ideas, but I'd first need to think through what you're actually asking for.
Why would you bring depreciation into the calculation which has no correlated feature/meaning ingame?
Unless you have to do with this IRL on a professional level, I don't see the value for a regular player/user, but I might be missing an important point and need more explanations on that. :)

As to the nanocrafting: you won't and shouldn't use the nanocrafter longer than needed, i.e. get your basic industries done (Assembler M can be bought on markets for cheap) and get them working in parallel (especially Pipes, Screws and all those fiddly parts) and get the talents queued.

**Comment by CCGLLC-knc** [comment]
Does ANY programmer like to document?  I haven't meet any, myself included.  Personally hate writing test code too...

Depreciation seems like a fair way to include the cost of the equipment used to manufacture the product.  Its not free, nor should it be all charged to the first item produced.  Depreciate the cost of the equipment (e.g. Industries in our case) over some reasonable number of products seems like a reasonable approach to me.  Its a counter to the "Hey, it only cost you 100,000 in ore why are you asking 120,000 for the product" questions.  Yeah, I may own the industry, but it wasn't free build it.  Short form: The cost of a product should include a portion of the cost of the gear required to manufacture it.

Re: Nanocrafting.  Admittedly this request would see the most use by beginners starting their bootstrapping.  During the recent PTS RC1.0 run I figured out Hydraulics pretty quickly, but missed Pipes as part of the critical path for awhile.  By the end of the PTS I was only using my nanocrafter for the one-off items needed to finish off my S-Core plant (like the specialty parts needed to build a 3D-Printer).

**Comment by tobitege** [comment]
Ah, I see what you mean. With depreciation I understood it more in the accounting sense of diminished book value of items over a certain amount of years (if my English doesn't fail me here). :)

**Comment by CCGLLC-knc** [comment]
If memory serves, "Depreciation" is applied to capital assets.  "Amortization" is applied to expenses.  Both happen over a period greater than a year and both are mostly used for tax purposes.  Still, the concept applies towards the cost of goods.  And your English is perfect!
